doc/nrf/libraries/bluetooth_services/scan.rst

```diff
@@ -8,78 +9,79 @@ Scanning module
    :depth: 2
 
 The scanning module handles the Bluetooth® Low Energy scanning for your application.
-You can use it to find advertising devices and establish connections with them.
 
-Using :ref:`filters <nrf_bt_scan_readme_filters>`, you can narrow down the scan to devices of a specific type.
+Overview
+********
 
-The Scanning Module can work in one of the following modes:
+Use the scanning module to find advertising devices and establish connections with them.
+The module can automatically establish connection after a filter match.
+It can also receive :ref:`directed advertising packets <lib_nrf_bt_scan_readme_directedadvertising>`.
 
-* Simple mode - without using filters
-* Advanced mode - that allows using advanced filters
+The scanning module can work in one of the following modes:
 
-The module can automatically establish a connection after a filter match.
-It can also receive :ref:`directed advertising packets <nrf_bt_scan_readme_directedadvertising>`.
+* Simple mode - That works without using filters.
+* Advanced mode - That allows using advanced filters.
 
-Usage
-*****
-
-
-You can use the scanning module to execute the following actions:
+Configuration
+*************
 
-Initialize
-   To initialize the module, call the function :c:func:`bt_scan_init`.
+To initialize the module, call the :c:func:`bt_scan_init` function.
```

Comment:
Can this description be used in Configuration section, or there's something missing here? Not sure if initializing the module is equal to configuring the module.

Reply:
No, I think this sentence belongs to the "Usage" section. I think "Usage" is how you use the library API in your application (i.e. when you are writing code).
"Configuration" is how you select the library to be part of your application (before you start any coding). this is done by selecting configuration options during using Kconfig tool which is part of the build system.
So I think, the following Kconfig options should go to this section: CONFIG_BT_SCAN - which enables the library in the build system, CONFIG_BT_SCAN_CONN_ATTEMPTS_FILTER, CONFIG_BT_SCAN_CONN_ATTEMPTS_FILTER_LEN, CONFIG_BT_SCAN_CONN_ATTEMPTS_COUNT etc. of the later sections from the old format: "Blocklist" and "Directed Advertising".

Reply:
Corrected, please check :)

Reply:
Looks good now.